### DISCUSSION TITLE
Regarding Gssoc

### DISCUSSION THREAD

**Discussion by Anuj3553** [Bitbox-Connect/Bitbox#77] [General]
Solve issues regarding Gssoc problem

**Comment by Anuj3553** [comment]
Hello, @sanjay-kv, @AKSHITHA-CHILUKA and @MastanSayyad

Can we discuss the problem here because i have also problem to look of your messages.

**Reply by MastanSayyad** [reply]
Yeah, sure! For now, there are none.

Just make sure to add appropriate labels to issues, assign issues, and merge contributors' PRs while staying active. @Anuj3553

**Reply by Anuj3553** [reply]
Okk understood.
